libcxx/test/std/time/time.zone/time.zone.leap/nonmembers/comparison.pass.cpp

```diff
@@ -10,7 +10,7 @@
 // UNSUPPORTED: no-filesystem, no-localization, no-tzdb
 
 // TODO TZDB test whether this can be enabled with gcc 14.
-// UNSUPPORTED: gcc-13
+// UNSUPPORTED: gcc-13, gcc-14
```

Comment:
Did you test this with GCC 14? If yes please update the comment.

Reply:
I tested it insofar that the test is still failing. I haven't looked closely at why. If you want to I can give you the output to check that it fails as expwcted.